src/actions/CryptoExchangeActions.js

```diff
@@ -281,6 +281,13 @@ const getShiftTransaction = (fromWallet: GuiWallet, toWallet: GuiWallet, info: S
     throw quoteError
   }
 
+  // totle and other DEX's may not allow swapping to a different wallet (received coins must come to same wallet)
+  if (edgeCoinExchangeQuote.pluginName === 'totle') {
```

Comment:
Plugins that have this requirement should declare that fact in their `PluginInfo` in the core, and then the GUI should limit the wallet picker accordingly. However, I realize we don't have time for that this late in the release cycle, so this will do the trick for now.